omnigibson/__init__.py

```diff
@@ -30,8 +30,6 @@
 
 __version__ = "1.0.0"
 
-log.setLevel(logging.DEBUG if gm.DEBUG else logging.INFO)
```

Comment:
why was this removed again?

Reply:
because gm is being accessed too early right?

Reply:
Yes! This is moved to the start of `_launch_app`. Please let me know if that make sense.